FixMyBerlin/fixmy.frontend
# Add new report statuses and update marker images

- Add new marker image assets
- Handle new report statuses inactive, planning, execution, tender, invalid
- Handle old report statuses if prefixed with "report_"

For https://github.com/FixMyBerlin/fixmy.platform/issues/323

## Type of change

- New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

1. Open marker overview map for berlin and aachen region, verify marker images
2. Open report detail panel

## Checklist:
